### ISSUE TITLE
Settings: bring back Accessibility section

### ISSUE THREAD

**Issue by pilgrim-brave** [brave/brave-browser#7050]
Upstream Chromium has a short but useful Accessibility subpage (`chrome://settings/accessibility`) with links to OS-level captions preferences and a curated list of related extensions in the web store. We removed this section long ago, but we should bring it back.

**Comment by pitsi** [comment]
Sorry for bringing this back up after 2+ years, but is there any chance of fixing it? I came accross a video player that uses captions to display subtitles which were to small for me and I wanted to make them bigger.

Then I noticed that not only the captions menu (the equivalent of chrome://settings/captions) is missing, but the entire accessibility menu (the equivalent of chrome://settings/accessibility) is missing too. And I was ready to file a new issue report, but I found this one first.

**Comment by Rogue-Git-Dev** [comment]
To this day the accessibility section is not existent in Brave's settings. This must change. Brave must incorporate more accessibility in the very least to have feature parity with Chromium.

**Comment by rebron** [comment]
Closing as dupe. Fixed in 1.78.x https://github.com/brave/brave-browser/issues/15844

**Comment by pitsi** [comment]
I do not want to sound mean, but the captions option (i.e. the reason I posted here) is still missing and `brave://settings/captions` still returns a blank settings page.


Chromium 139.

<img width="700" height="410" alt="Image" src="https://github.com/user-attachments/assets/681bda3d-aaa2-4be3-b498-172087af3971" />

Brave 1.81.

<img width="700" height="350" alt="Image" src="https://github.com/user-attachments/assets/ac2b0d5a-4ffb-41a3-b8d2-154c863824c7" />

---edit
Also confirmed with a couple of windows users.
